tests/test_math.py

```diff
@@ -1,9 +1,17 @@
-from elk.math_util import stochastic_round_constrained
+from elk.math_util import batch_cov, cov_mean_fused, stochastic_round_constrained
 from hypothesis import given, strategies as st
 from random import Random
 import math
 import numpy as np
 import pytest
+import torch
+
+
+def test_cov_mean_fused():
+    X = torch.randn(10, 500, 100, dtype=torch.float64)
+    cov_gt = batch_cov(X).mean(dim=0)
+    cov_fused = cov_mean_fused(X)
+    assert torch.allclose(cov_gt, cov_fused)
 
```

Comment:
yeahhh we should probably make this opt-out rather than opt-in. Would be awesome if you could make a PR for that at some point :)

Will add the `pytest.mark.cpu` in the meantime though